contrib/terraform/upcloud/cluster-settings.tfvars

```diff
@@ -0,0 +1,58 @@
+
+# See: https://developers.upcloud.com/1.3/5-zones/
+zone   = "fi-hel1"
```

Comment:
Please run `terraform fmt` and add a CI/CD job like here: https://github.com/kubernetes-sigs/kubespray/blob/master/.gitlab-ci/terraform.yml#L104